gmat/tests/template_frames.script

```diff
@@ -0,0 +1,113 @@
+%----------------------------------------
```

Comment:
Let's rename this to `template_frames.script.tpl`, so it's clear it's a template.

Reply:
Renaming... ✍🏽